SonarQube.Bootstrapper/Resources.resx

```diff
@@ -148,7 +145,7 @@
     <value>Downloading {0} from {1} to {2}</value>
   </data>
   <data name="INFO_ProcessingStarted" xml:space="preserve">
-    <value>{0} ({1} argument(s) passed)</value>
+    <value>{0} started.</value>
```

Comment:
We can't easily get the number of arguments any more, and in any event the process runner already logs it.